ratbagd/ratbagd-profile.c

```diff
@@ -528,6 +528,11 @@ int ratbagd_profile_new(struct ratbagd_profile **out,
 		if (!led)
 			continue;
 
+		/* We don't care about LEDs that we can't change the color
+		 * for */
+		if (ratbag_led_get_colordepth(led) == RATBAG_LED_COLORDEPTH_MONOCHROME)
+			continue;
```

Comment:
Nack

On the G502 non rgb, a quite popular mouse, setting the LED is important. Because the LED is blue and I remember not being able to sleep in the same room than this mouse because of the default breathing effect. I ended up unplugging the mouse IIRC, but this setting should still be presented to the user.

IMO, the Color dbus parameter should not be exposed, but the mode and brightness should

Reply:
removed from the patchset